cmd/kratos/internal/proto/client/client.go

```diff
@@ -16,8 +16,8 @@ import (
 // CmdClient represents the source command.
 var CmdClient = &cobra.Command{
 	Use:   "client",
-	Short: "Generate the proto client code",
-	Long:  "Generate the proto client code. Example: kratos proto client helloworld.proto",
+	Short: "Generate the proto Client code",
+	Long:  "Generate the proto Client code. Example: kratos proto client helloworld.proto",
```

Comment:
Why change to uppercase?

Reply:
Smooth and consistent with Server

Reply:
This is just a hint. `use` is  `client`?